README.md

```diff
@@ -1,5 +1,10 @@
 <h1 align="center">NoMoreDomains</h1>
 
+<div align="center">
+    <img alt="Desktop" title="#Desktop" src="./assets/mock.jpg" width="680px" />
```

Comment:
this mock-up is not what we need, it does not show an accurate representation of this tool.